README.md

```diff
@@ -1,3 +1,44 @@
+# Micro-Reddit
+This is a project built based on the Active Record (Model) principle. It is built with Ruby on Rails frame work.
```

Comment:
OPTIONAL: You could add a brief description of the project features here - the models and associations you have created.